libs/blocks/tabs/tabs.js

```diff
@@ -68,6 +68,11 @@ function configTabs(config, rootElem) {
     const sel = rootElem.querySelector(id);
     if (sel) sel.click();
   }
+  const tabParam = new URLSearchParams(window.location.search).get('tab');
+  if (!tabParam) return;
+  const dashIndex = tabParam.lastIndexOf('-');
+  const [tabsId, tabIndex] = [tabParam.substring(0, dashIndex), tabParam.substring(dashIndex + 1)];
```

Comment:
Should we strengthen the checks here for `dashIndex`'s value? It might be that this is authored erroneously and errors will pop up, such as `tab2`.